test/decl/class/actor/basic.swift

```diff
@@ -2,12 +2,20 @@
 
 // REQUIRES: concurrency
 
-actor class MyActor { }
+actor MyActor { }
 
 class MyActorSubclass1: MyActor { }
 
-actor class MyActorSubclass2: MyActor { }
+actor MyActorSubclass2: MyActor { }
+
+// expected-warning@+1:7{{'actor class' has been renamed to 'actor'}}{{7-13=}}
+actor class MyActorClass { }
 
 class NonActor { }
 
-actor class NonActorSubclass : NonActor { } // expected-error{{actor class cannot inherit from non-actor class 'NonActor'}}
+actor NonActorSubclass : NonActor { } // expected-error{{actor class cannot inherit from non-actor class 'NonActor'}}
```

Comment:
should we instead be calling them "actor" rather than actor class in all diagnostics from here on?

Reply:
Yeah, that seems like a good idea.